services/asset/src/lib.rs

```diff
@@ -144,6 +134,16 @@ impl<SDK: ServiceSDK> AssetService<SDK> {
         }
     }
 
+    #[cycles(10_000)]
+    #[read]
+    fn get_admin(&self, ctx: ServiceContext, asset_id: Hash) -> ServiceResponse<Address> {
+        require_asset_exists!(self, asset_id);
+
+        // It is impossible to panic!
```

Comment:
Yes it is! I know that's shouldn't be None.
However, to keep a standard, pls /match/ it and return ServiceResponse with error code.
That's the best flavour :)

Reply:
The reason why I wrote in this way is to maintain the same style as Zeroqn.